setup.py

```diff
@@ -32,7 +32,12 @@ def get_snakemake_pkg(extras=""):
     "docs": ["Sphinx>=1.4.4", "sphinx-rtd-theme>=0.1.9",],
     "tests": tests_require,
     "kubernetes": ["kubernetes>=11.0.0,<12.0.0",],
-    "yadage": ["adage~=0.10.1", "yadage~=0.20.1", "yadage-schemas~=0.10.6"],
+    "yadage": [
+        "adage~=0.10.1",
+        "yadage~=0.20.1",
+        "yadage-schemas~=0.10.6",
+        "pygraphviz<1.8",
```

Comment:
[`snakemake[reports]` also installs this package](https://github.com/snakemake/snakemake/blob/5fabffbb4af8e9e122677e5adeaebf2d6bd0eeb3/setup.py#L81) so it probably should be at the `install_requires` level.

Reply:
The only thing is that if nor `snakemake`, nor `yadage` are installed, `pygraphviz` still will be still installed.

Reply:
Let's add it only as part of `snakemake_reports`.

Reply:
Although, this is only part of the `reports` extra, and that's only being used in the workflow engine, that runs py38, so we should be safe without it...

Reply:
Don't we need the pin also in the `yadage` extra?

Reply:
`pygraphviz` in `yadage` is already pinned ([source](https://github.com/yadage/adage/blob/353c15cd97ff5150eff128f34cf1666c78826524/setup.py#L7)). CI for python 3.6 and 3.7 passing.

Reply:
Ah good, in addition, that's the `viz` extra and we don't install it.